csharp/src/Ice/PropertiesI.cs

```diff
@@ -10,20 +10,19 @@ internal sealed class PropertiesI : Properties
 {
     private class PropertyValue
```

Comment:
can we use a record class here too?

Reply:
I don't think it offers much over a regular class here.